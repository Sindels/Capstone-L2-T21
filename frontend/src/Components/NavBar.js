import React from "react";
import {Nav } from "react-bootstrap";
import "../App.css"; //import the stylesheet, react and Nav from bootstrap

function NavBar() {
  return (
    <header>
      <Nav className="navbar">
        {/*---Logo icon image --- */}
        <a href="/" className="brand">
          <div >
            <h2> iTunes Search API</h2> {/*When clicking on this header it will return you to the default page*/}
          </div>
        </a>
      </Nav>
      
    </header>
  );
}

export default NavBar; //export NavBar component