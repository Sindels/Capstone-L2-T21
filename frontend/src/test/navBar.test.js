import React from 'react';
import NavBar from '../Components/NavBar';
import renderer from 'react-test-renderer'; //imported react, renderer and the NavBar component 

test('Renders correctly', () => {
    const tree = renderer
        .create(<NavBar/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
}); //This will test if the Navbar component Renders correctly without any errors and create a snapshot test