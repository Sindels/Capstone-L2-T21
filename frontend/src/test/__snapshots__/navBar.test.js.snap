// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<header>
  <div
    className="navbar nav"
    onKeyDown={[Function]}
  >
    <a
      className="brand"
      href="/"
    >
      <div>
        <h2>
           iTunes Search API
        </h2>
         
      </div>
    </a>
  </div>
</header>
`;
